src/patch.rkt

```diff
@@ -201,14 +201,9 @@
                   (loop (cdr cl) (alt prog* (list 'change (car cl)) (list altn)))
                   done))))))
 
-  (define rewritten*
-    (if (and (*pareto-mode*) (> (length rewritten) 1000))
-        (take rewritten 1000)
-        rewritten))
```

Comment:
Not sure why this was removed, but I'm happy it is.

Reply:
Artifact from the paper.